app/views/pages/history.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="page-container">
+  <%= stylesheet_link_tag "home" %>
+  <h1>History</h1>
+  <br>
+  <h4>Issues</h4>
+  <br>
+  <% if current_user %>  
+    <% if current_user.issues.any? %>
+      <div>You have <%= pluralize(current_user.issues.length, 'issue')  %>:</div>
+      <%= render 'issues/show_list', issues: current_user.issues  %>
+    <% else %>
+      <strong>You have not reported any issues yet!</strong>
+    <% end %>
+  <% else %> 
+    <div>Login to see the history</div>
```

Comment:
Per my other comment below, I'd change this to "Login to see your reported issues."

Reply:
ok